content/_index.md

```diff
@@ -31,20 +30,20 @@ codes:
 
 
       # Generate two large arrays
-      a = np.random.randint(0,2**32,2**10)
-      b = np.random.randint(0,2**32,2**10)
+      a = np.random.randint(0,2**32,2**28) # ----> smaller to fit on a single machine
+      b = np.random.randint(0,2**32,2**28) # /
```

Comment:
Empty comment can be removed

Reply:
The comment is not empty (it has a `/` for an arrow) but I can remove it anyway.